concepts/select/about.md

```diff
@@ -0,0 +1,142 @@
+# The SELECT statement
+
+In SQL, a `SELECT` statement allows you to retrieve data from a database.
+The result of a `SELECT` statement is a result set -- some number of values arranged in rows and columns, where each column is a particular attribute of the data and each row is a set of values for each column.
+
+Roughly, you can think of the set of columns as a data structure, where each column represents a field of the data structure, and each row represents a concrete instance of the data structure.
+
+With a `SELECT` statement, you can specify the data you want and optionally transform, filter, and/or modify the shape of the output data.
+
+
+## The basics
+
+The anatomy of a basic `SELECT` statement is as follows:
+
+```sql
+SELECT <columns>
+FROM <source>
+WHERE <criteria>;
```

Comment:
If this is the first time someone is seeing a SELECT, it would be good to make it clear that the `WHERE` is optional.

Reply:
I'm not a fan of the brackets. I would suggest either (1) showing three examples with the different options, or (2) having a "Note" line immediately after the codeblock (making it more urgent/visible) and not inside the exercism/note box; I think it would be more visible at the top level. It might be cleaner if the Hello World output block was dropped.

Reply:
I kept it in an exercism/note box, but made it more immediate, and made the note about whitespace subordinate.  I also removed the Hello World output as suggested.  Let me know what you think about the new version.